src/main/java/education/cursor/movies/service/MovieService.java

```diff
@@ -47,4 +56,8 @@ public Movie update(Movie movieFromDb, Movie movie) {
 
         return movieRepository.save(movieFromDb);
     }
-}
+
+    public List<Movie> getAllMovieByCategory(Movie movie) {
```

Comment:
Invalid method arguments. Look in the controller above I have explained the reason.